docs/mkdocs/docs/index.md

```diff
@@ -1,387 +1,495 @@
 
 ```python
->>> ac = adb.Arctic('s3s://ENDPOINT:BUCKET?aws_auth=true')
+library = ac.get_library('data', create_if_missing=True)
 ```
 
-!!! s3 vs s3s
+!!! info "ArcticDB Static & Dynamic Schemas"
 
-    Use `s3s` if your S3 endpoint used HTTPS
+    ArcticDB does not need data schemas, unlike many other databases. You can write any DataFrame and read it back later. If the shape of the data is changed and then written again, it will all just work. Nice and simple.
 
-##### Connecting to a defined storage endpoint
+    The one exception where schemas are needed is in the case of functions that modify existing symbols: `update` and `append`. When modifying a symbol, the new data must have the same schema as the existing data. The schema here means the index type and the name, order, and type of each column in the DataFrame. In other words when you are appending new rows they must look like the existing rows. This is the default option and is called `static schema`.
 
-Connect to local storage (not AWS - HTTP endpoint of s3.local) with a pre-defined access and storage key:
+    However, if you need to add, remove or change the type of columns via `update` or `append`, then you can do that. You simply need to create the library with the `dynamic_schema` option set. See the `library_options` parameter of the ([`create_library`](api/arctic/#arcticdb.Arctic.create_library)) method.
+
+    So you have the best of both worlds - you can either enforce a static schema on your data so it cannot change by modifying operations, or allow it to be flexible.
+    
+    The choice to use static or dynamic schemas must be set at library creation time.
+
+    In this section we are using `static schema`, just to be clear.
+
+### Reading And Writing Data
+
+Now we have a library set up, we can get to reading and writing data. ArcticDB exposes a set of simple functions to enable DataFrame storage. 
+
+Let's first look at writing a DataFrame to storage:
 
 ```python
->>> ac = adb.Arctic('s3://s3.local:arcticdb-test-bucket?access=EFGH&secret=HGFE')
+# 50 columns, 25 rows, random data, datetime indexed.
+import pandas as pd
+import numpy as np
+from datetime import datetime
+cols = ['COL_%d' % i for i in range(50)]
+df = pd.DataFrame(np.random.randint(0, 50, size=(25, 50)), columns=cols)
+df.index = pd.date_range(datetime(2000, 1, 1, 5), periods=25, freq="H")
+df.head(5)
+```
+_output (the first 5 rows of the data)_
+```
+                     COL_0  COL_1  COL_2  COL_3  COL_4  COL_5  COL_6  COL_7  ...
+2000-01-01 05:00:00     18     48     10     16     38     34     25     44  ...
+2000-01-01 06:00:00     48     10     24     45     22     36     30     19  ...
+2000-01-01 07:00:00     25     16     36     29     25      9     48      2  ...
+2000-01-01 08:00:00     38     21      2      1     37      6     31     31  ...
+2000-01-01 09:00:00     45     17     39     47     47     11     33     31  ...
```

Comment:
As a user I find it better when the examples are in a single snippet that I can copy/paste, rather than having to search the docs for the relative pieces and stich something working myself. I'd make one complete example creating a dataframe, creating arctic instance and library, storing the symbol and reading it back all in once place. And place the expected output below that.